### DISCUSSION TITLE
audio resources - enter file information

### DISCUSSION THREAD

**Discussion by aaronjnorris** [bonsai-rx/bonsai#1333] [General]
I am wondering if it is user error or a bug.  I was building off the examples in this Google thread [https://groups.google.com/g/bonsai-users/c/i_4wM-qigPY/m/umpoE7d3CwAJ](url) and see in those there is a way to input a file location in the nodes in those versions but if I created a new audioresource node I could not input a file location.
![image](https://user-images.githubusercontent.com/29385322/234339734-5a77808b-561c-4235-8d08-4d288aa33088.png)
![image](https://user-images.githubusercontent.com/29385322/234339810-15c2009f-99cd-42d3-aae6-11e947af40ac.png)

Is this just a user issue and just my bonsai install? How can input the file and location to load in a new node?
[sound-An.zip](https://github.com/bonsai-rx/bonsai/files/11324703/sound-An.zip)

**Comment by bruno-f-cruz** [comment]
Add an `AudioResources` operator, select the `Buffers` property,  click the arrow on the right side of the `Add` button and select `SoundBuffer`.

**Comment by aaronjnorris** [comment]
Ok.....I didn't have the arrow next to the add button but uninstalling the audio library and reinstalling fixed this.  Thanks!

**Reply by horsto** [reply]
> Ok.....I didn't have the arrow next to the add button but uninstalling the audio library and reinstalling fixed this. Thanks!

I ran into the same issue. I tried deleting the package folder `Bonsai.Audio.2.7.0` and re-installing the audio library, but no luck. Any ideas?

**Reply by horsto** [reply]
Ok, Installing `Bonsai - System Design Library` solved it (https://packages.nuget.org/packages/Bonsai.System.Design/2.7.0).
